### PR TITLE
Send passcode reset email

### PR DESCRIPTION
**Pull request by sumesh-aot**
*Issue #:*
https://github.com/bcgov/entity/issues/6222

*Description of changes:*
- Changes for passcode reset event and email

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

**Review by karthik-aot — APPROVED** [review]

File: queue_services/account-mailer/q_cli.py [hunk 1]
```diff
@@ -220,56 +106,23 @@ def subscription_options():
...
+    event_loop.run_until_complete(run(event_loop, payload, replace_payload))
 
-# pad cmd --> python3 q_cli.py -m pad -i 10 -n TestAccount -b 088 -t 00277 -a 12874890
-# refund cmd --> python3 q_cli.py -m refund -i 10 -o 67892 -p 25.33 -d 988
-# account suspended cmd --> python3 q_cli.py -m acc_lock -i 4 -n SomeAccount
-# account restored cmd --> python3 q_cli.py -m acc_unlock -i 4 -n SomeAccount
-# account conf pver-- >  python3 q_cli.py -m conf_per -i 4
-# account conf pver-- >  python3 q_cli.py -m pad_invoice -i 4
-# account admin removal-- >  python3 q_cli.py -m admin_removed -i 4
+# Examples:
```
Comment (saravanpa-aot) [inline queue_services/account-mailer/q_cli.py:127]: do you mind adding one more sample here? 
mostly many non-python devs use this to copy the command.. just one sample will be helpful

File: queue_services/account-mailer/q_cli.py [hunk 2]
```diff
@@ -85,133 +83,21 @@ def subscription_options():
...
-                'id': auth_account_id,
-                'datacontenttype': 'application/json',
-                'data': {
-                    'accountId': auth_account_id,
-                    'cfsAccountId': cfs_account_id,
-                    'transactionAmount': transaction_amount
-                    # 'recipientEmail': 'foo@bar.com'
-                }
-            }
+        payload_template: dict = {
```
Comment (saravanpa-aot) [inline queue_services/account-mailer/q_cli.py:86]: great clean up

**Review by saravanpa-aot — APPROVED** [review]

File: queue_services/account-mailer/src/account_mailer/enums.py [hunk 3]
```diff
@@ -48,6 +49,7 @@ class SubjectType(Enum):
     PAD_SETUP_FAILED = '[BC Registries and Online Services] Your Account is Temporarily Suspended'
     PAYMENT_PENDING = '[BC Registries and Online Services] Payment is now due for pending transaction on your account'
     EJV_FAILED = 'GL disbursement failure for EJV'
+    RESET_PASSCODE = 'Passcode reset'
```
Comment (saravanpa-aot) [inline queue_services/account-mailer/src/account_mailer/enums.py:52]: do you wanna add the [BC Registries and Online Services] in the header?

Comment (sumesh-aot) [reply queue_services/account-mailer/src/account_mailer/enums.py:52]: I have asked subject and some header details from Roy in the ticket. Once he confirms, will change it. What I put there is only dummy.

File: queue_services/account-mailer/q_cli.py (hunk 1 above)
Comment (sumesh-aot) [reply queue_services/account-mailer/q_cli.py:127]: Done

**Pull request merged**